vignettes/safestats-vignette.Rmd

```diff
@@ -188,30 +186,30 @@ Due to sampling error, the average number of times that $S > 1 / \alpha$ might n
 
 ## <a name = "optStopT">Safe Tests Allow for Optional Stopping without Inflating the Type I Error Rate above the Tolerable $\alpha$-Level</a>
 <!--### Data under the alternative: <a name = "optStopT">Stopping the experiment early</a>-->
-What makes safe tests particularly interesting is that they allow for early stopping without the test exceeding the tolerable type I error rate of $\alpha$. This means that the evidence can be monitored as the data comes in, and when there is a sufficient amount of evidence against the null, thus, $S > 1/\alpha$, the experiment can be stopped early, which therefore increases efficiency. We highlight this with our design object 
+What makes the safe tests in this package particularly interesting is that they allow for early stopping without the test exceeding the tolerable type I error rate of $\alpha$. This means that the evidence can be monitored as the data comes in, and when there is a sufficient amount of evidence against the null, thus, $S > 1/\alpha$, the experiment can be stopped early, which therefore increases efficiency.
+
+> Note that <a name = "martingale">not all s-values necessarily allow for optional stopping</a>: this only holds for some special s-values, that are also _test martingales_. More information can be found, for example, in the first author's [master thesis](https://www.universiteitleiden.nl/binaries/content/assets/science/mi/scripties/statscience/2019-2020/thesis_rjturner_for_publication.pdf), Chapter 5.
+
+We highlight this with our design object 
```

Comment:
By "highlighting", I meant that I want to use the same design object as derived above to clarify optional stopping. This wasn't really clear, perhaps this is better:

    For this purpose, we use the design that was derived above, that is, 

```{r}
designObj
```